### PR TITLE
Conditional Automation in Driving: Level 3 Autonomous Cars

### LINKED ISSUE #674: Conditional Automation in Driving: Level 3 Autonomous cars
### Brief Summary: 
Autonomous driving is gaining momentum in the automobile industry. This area has gained interest among developers because of the application of artificial intelligence and augmented reality (AR) in autonomous driving. This article aims at providing readers with meaningful information regarding the third level of autonomous driving.

### Key Takeaways: 
1. What is an autonomous car?
2. Introduction to conditional automation in driving
3. Key features of a level 3 autonomous car
4. How level 3 autonomous cars work
5. The future of autonomous driving

### References: 
https://interestingengineering.com/how-do-self-driving-cars-work

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

This PR fixes #674 .

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published. 
Is this PR ready to be reviewed? 🤔  Just wondering becuase of all the commits

**Comment by hectorkambow** [comment]
Just an FYI - moving forward - lets please only create a PR once the .md file is fully ready to be reviewed - any changes can be made on your personal forked repo.

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/conditional-automation-in-driving/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+In level 3 cars, the human driver is simply a co-driver who will only take control under certain conditions. The driver can engage in other things like watching the surroundings or texting. The autonomous car will handle situations that require an immediate response. For example, when another vehicle suddenly appears in front of the car, the vehicle will perform emergency braking.
+
+### Key Features of a Level 3 Autonomous Car
+A level 3 autonomous car has certain technological features that facilitate self-driving. 
+
+#### Computer Power
+A computer system is required in a level 3 car to enhance the processing of information and activities within the car. Extreme computing power is required to support the automation of controls (more than the traditional CPUs).
+
+Graphical processing units (GPUs) are used by automobile companies for processing. However, GPUs have not been effective enough to provide the sufficient processing power needed in self-driving cars. This made Tesla to use neural network acceleration chips (NNA), which has a higher processing capacity than GPU.
```
Comment (lalith1403) [inline articles/conditional-automation-in-driving/index.md:34]: Can you include links to NNAs? Also aren't FPGAs used. They are power efficient and better at computations than GPUs.

File: articles/conditional-automation-in-driving/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+### Key Features of a Level 3 Autonomous Car
+A level 3 autonomous car has certain technological features that facilitate self-driving. 
+
+#### Computer Power
+A computer system is required in a level 3 car to enhance the processing of information and activities within the car. Extreme computing power is required to support the automation of controls (more than the traditional CPUs).
+
+Graphical processing units (GPUs) are used by automobile companies for processing. However, GPUs have not been effective enough to provide the sufficient processing power needed in self-driving cars. This made Tesla to use neural network acceleration chips (NNA), which has a higher processing capacity than GPU. 
+
+#### Cameras
+Cameras are used to capture the road and surrounding environments. The captured images help the autonomous car to choose the right control depending on the situation. For example, if the camera senses an image in front of the car, a braking control will be initiated. The visual data collected using cameras is translated into 3D data that is actionable.
```
Comment (lalith1403) [inline articles/conditional-automation-in-driving/index.md:37]: The visual data collected using cameras is translated into 3D actionable data.

File: articles/conditional-automation-in-driving/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+#### Computer Power
+A computer system is required in a level 3 car to enhance the processing of information and activities within the car. Extreme computing power is required to support the automation of controls (more than the traditional CPUs).
+
+Graphical processing units (GPUs) are used by automobile companies for processing. However, GPUs have not been effective enough to provide the sufficient processing power needed in self-driving cars. This made Tesla to use neural network acceleration chips (NNA), which has a higher processing capacity than GPU. 
+
+#### Cameras
+Cameras are used to capture the road and surrounding environments. The captured images help the autonomous car to choose the right control depending on the situation. For example, if the camera senses an image in front of the car, a braking control will be initiated. The visual data collected using cameras is translated into 3D data that is actionable. 
+
+#### Radar
+The level 3 car uses radar to detect the distance and presence of objects that are in front of the car. Radar helps the car to achieve control in times of snow or rain.
```
Comment (lalith1403) [inline articles/conditional-automation-in-driving/index.md:40]: Some more information regarding extreme conditions is required. When do cameras fail. How much of the work is carried out by cameras and how much by radars and other sensors? What conditions are ideal? How are edge cases overcome. Answer all these questions.

File: articles/conditional-automation-in-driving/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+#### Cameras
+Cameras are used to capture the road and surrounding environments. The captured images help the autonomous car to choose the right control depending on the situation. For example, if the camera senses an image in front of the car, a braking control will be initiated. The visual data collected using cameras is translated into 3D data that is actionable. 
+
+#### Radar
+The level 3 car uses radar to detect the distance and presence of objects that are in front of the car. Radar helps the car to achieve control in times of snow or rain. 
+
+#### Sensors
+Various sensors are used to detect situations and respond to them. For example, in case of failure, the car will use sensors to detect it, which will prompt the driver to take control. LIDAR sensors are placed at the top of the car to provide 3D data on the area surrounding the car. Inertial sensors, ultrasonic sensors, and GPS tracking sensors are utilized to monitor the performance of the autonomous car.
```
Comment (lalith1403) [inline articles/conditional-automation-in-driving/index.md:43]: what failure? specify the type of failure.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
An interesting article. I had a few questions as a reader that should be answered to deliver supreme value to the reader. Ensure you answer those questions and perform a Grammarly check before submitting.

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/conditional-automation-in-driving/index.md (hunk 1 above)
Comment (mbaabuones) [reply articles/conditional-automation-in-driving/index.md]: I have included a link for the explanation of how NNA chips are used by Tesla. I have also added some information about FPGAs.

File: articles/conditional-automation-in-driving/index.md (hunk 2 above)
Comment (mbaabuones) [reply articles/conditional-automation-in-driving/index.md]: Done.

File: articles/conditional-automation-in-driving/index.md (hunk 3 above)
Comment (mbaabuones) [reply articles/conditional-automation-in-driving/index.md]: I have made changes accordingly.

**Comment by lalith1403** [comment]
Notify on all PRs once done. That way it makes it easier for us to proceed,

File: articles/conditional-automation-in-driving/index.md (hunk 4 above)
Comment (mbaabuones) [reply articles/conditional-automation-in-driving/index.md]: I have modified the sentence.

**Comment by mbaabuones** [comment]
> 
> 
> Notify on all PRs once done. That way it makes it easier for us to proceed,

Okay. Im done with this one.

**Comment by lalith1403** [comment]
@hectorkambow there is some possible merge conflict. Can you kindly check that? I think a duplicate PR might be needed. Not sure.

**Comment by hectorkambow** [comment]
@mbaabuones There is a merge conflict with this PR (not sure why - it says that the forward chaining article is still present) 

If you could please migrate the most updated .index file and all image to a new PR to be merged. (that should be a separate branch) 
Thank you

**Comment by mbaabuones** [comment]
> @mbaabuones There is a merge conflict with this PR (not sure why - it says that the forward chaining article is still present)
> 
> If you could please migrate the most updated .index file and all image to a new PR to be merged. (that should be a separate branch)
> Thank you

@hectorkambow Done.

**Comment by hectorkambow** [comment]
duplicate of #767

**Pull request closed without merging**